README.md

```diff
@@ -15,3 +18,13 @@ Go to the project root directory.
 git config user.name "John Doe"
 git config user.email "john.doe@mail.com"
 ```
+
+2. Create virtual environment and install project dependencies using Poetry.
```

Comment:
<details><summary>Opinionated suggestion</summary>
<p>

I would like to make it less verbose:

> 2. Create virtual environment and install ~project~ dependencies ~using Poetry~.

The less words our guide has then it's easier for someone to follow it, and it's easier for us to keep it consistent.

</p>
</details>